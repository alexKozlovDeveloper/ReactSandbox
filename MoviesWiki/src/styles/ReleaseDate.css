.releasedate {
    float: right;  
    color: white;  
    padding: 3px;
}