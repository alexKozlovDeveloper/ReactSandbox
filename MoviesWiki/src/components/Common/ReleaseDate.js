import React, { Component } from 'react';

import styles from '../../styles/ReleaseDate.css';

const ReleaseDate = ({ releaseDate }) => <div className={styles.releasedate}>
        {releaseDate}
    </div>;

export default ReleaseDate;
