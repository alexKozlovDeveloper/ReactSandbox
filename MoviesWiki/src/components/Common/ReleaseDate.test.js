import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ReleaseDate from './ReleaseDate';

configure({ adapter: new Adapter() });

test('ItemReleaseDate rendering test', () => {
  const releaseDate = 'someReleaseDate';

  const component = shallow(<ReleaseDate releaseDate={releaseDate} />);
  expect(shallowToJson(component)).toMatchSnapshot();
});
