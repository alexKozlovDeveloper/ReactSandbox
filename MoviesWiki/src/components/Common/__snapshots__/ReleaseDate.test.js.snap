// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemReleaseDate rendering test 1`] = `
<div
  className="releasedate"
>
  someReleaseDate
</div>
`;
